pyemu/utils/pst_from.py

```diff
@@ -178,20 +165,10 @@ def parfile_relations(self):
                 else pd.Series(
                     {k: v for k, v in [['lbound', x.lower_bound]]}), axis=1)
             if lbound.nunique(0, False).gt(1).any():
-                if lbound.nunique(0, False).gt(2).any():
-                    self.logger.lraise(
-                        "different lower bounds requested for same par for {0}"
-                        "".format(name))
-                else:
-                    fil = lbound.apply(lambda x:
-                                       pd.Series([None]) if x.isna().all()
-                                       else x[x.notna()].values).T
-                    self.logger.warn("Lower bound for par passed for some but "
-                                     "not all instances, will set NA to "
-                                     "passed values\n{}".format(fil))
-                    pr.loc[g.index, 'lower_bound'] = g.use_cols.apply(
-                        lambda x: [fil[0].loc['lbound{0}'.format(c)] for c in x]
-                        if x is not None else fil[0].loc['lbound'])
+                lb_max = lbound.min().fillna(self.ult_lbound_fill).to_dict()
```

Comment:
Ditto above, and should that one be 'max()' for lbound?

Reply:
correct